service/http.ts

```diff
@@ -8,20 +11,44 @@ export interface HttpError {
   };
 }
 
-async function http<T>(
+interface HttpConfig {
+  init?: Omit<RequestInit, "body">;
+  omitAuthentication?: boolean;
+}
+
+let _unauthenticatedErrorHandler: (
+  e: HttpError
+) => Promise<void> = async () => {};
+export const setUnauthenticatedErrorHandler = (
+  f: (e: HttpError) => Promise<void>
+) => {
+  _unauthenticatedErrorHandler = f;
+};
```

Comment:
Does it still make sense to do this with this new HTTP client since we don't actually have any interceptors? Or at least it's not clear that pattern is applicable? That's literally all this was used for with the old client and was a bit of a hack tbh to allow something external to "tap inside" of React

Reply:
Ahh I didn't see you are actually using this in `authenticatedHttp` when I looked at this, which combined with what you said there, it makes sense considering our most global state is still effectively lives in React 👍